src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/12 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarnings1.sql

```diff
@@ -62,4 +65,23 @@ WHERE ae.EpisodeStartDate >= (
         ELSE CONVERT(VARCHAR(10), '07/31/' +  Cast(CalendarYear as varchar) , 101) END
         From  Reference.CollectionPeriods Where [Open] = 1)
 AND    ae.TransactionType In (1,2,3,4,5,6,7,8,9,10,11,12,15)
-GO
\ No newline at end of file
+GO
+
+IF EXISTS (
+		SELECT 1
+		FROM [sys].[indexes] i
+		JOIN sys.objects t ON i.object_id = t.object_id
+		WHERE t.name = 'ApprenticeshipEarnings1'
+		AND i.[name] = 'IX_ApprenticeshipEarnings1_UKPRN'
+		)
+BEGIN
+	DROP INDEX [IX_ApprenticeshipEarnings1_UKPRN]
+		ON [Staging].[ApprenticeshipEarnings1]
+END
+
+CREATE CLUSTERED INDEX [IX_ApprenticeshipEarnings1_UKPRN] ON [Staging].[ApprenticeshipEarnings1]
+(
+	[Ukprn] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, SORT_IN_TEMPDB = OFF, DROP_EXISTING = OFF, ONLINE = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+GO
+
```

Comment:
there is a step in the payments process that indexes these three tables and rather than vw_apprenticeshipearning looking at apprenticeshipearning1, 2, 3, it looks at a new table that contains all the data from the three tables.

Therefore, remove index.